app/app/src/main/kotlin/com/hedvig/app/feature/chat/ui/ChatTextInput.kt

```diff
@@ -76,9 +82,9 @@ class ChatTextInput : AbstractComposeView {
   override fun Content() {
     HedvigTheme {
       Surface(
-        color = if (isSystemInDarkTheme()) button_background_dark else hedvig_light_gray,
-        contentColor = contentColorFor(MaterialTheme.colorScheme.surface),
-        shape = MaterialTheme.shapes.medium,
+        color = MaterialTheme.colorScheme.surfaceVariant,
+        contentColor = contentColorFor(MaterialTheme.colorScheme.onSurfaceVariant),
+        shape = MaterialTheme.shapes.squircle,
```

Comment:
Might need to change this to
```suggestion
        shape = MaterialTheme.shapes.squircleMedium,
```